app/views/review/submit_2i_verdict.erb

```diff
@@ -23,26 +23,25 @@
   <%= render "shared/steps/form_errors", resource: @step_by_step_page %>
 <% end %>
 
-<%= form_tag do %>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <%= render "govuk_publishing_components/components/textarea", {
-        label: {
-          text: approved ? "Add a note (optional)" : "Describe changes required"
-        },
-        name: approved ? "additional_comment" : "requested_change",
-      } %>
-    </div>
-  </div>
+<%= render "govuk_publishing_components/components/contextual_guidance", {
+  html_for: "additional-comments",
+  title: approved ? t("step_by_step_pages.2i_review_approved.title") : t("step_by_step_pages.2i_review_request_changes.title"),
+  content: approved ? t("step_by_step_pages.2i_review_approved.guidance") : t("step_by_step_pages.2i_review_request_changes.guidance")
+} do %>
+  <%= form_tag do %>
+    <%= render "govuk_publishing_components/components/textarea", {
+      label: {
+        text: approved ? t("step_by_step_pages.2i_review_approved.label") : t("step_by_step_pages.2i_review_request_changes.label")
+      },
+      id: "additional-comments",
+      name: approved ? "additional_comment" : "requested_change",
+    } %>
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      <%= render "govuk_publishing_components/components/button", {
-        text: approved ? "Yes, approve 2i" : "Send change request"
-      } %>
-    </div>
-    <div class="govuk-grid-column-full govuk-!-margin-top-3">
-      <%= link_to "Cancel", @step_by_step_page, class: "govuk-link" %>
-    </div>
-  </div>
+    <%= render "govuk_publishing_components/components/button", {
+      text: approved ? "Yes, approve 2i" : "Send change request",
+      margin_bottom: true
+    } %>
+
+    <%= tag.p link_to("Cancel", @step_by_step_page, class: "govuk-link"), class: "govuk-body" %>
+  <% end %>
```

Comment:
There's a lot of small conditionals here that make this somewhat hard to read.

I wonder if the whole thing should be in a conditional.

For example:

```
<% if approved %>
  <%= render "govuk_publishing_components/components/contextual_guidance" .. %>
<% else %>
     <%= render "govuk_publishing_components/components/contextual_guidance" .. %>
<% end %>
```

or set some variables first,

for example:

```
phrase_prefix = "step_by_step_pages.2i_review_#{approved ? 'approved' : 'request_changes'}"
textarea_name = approved ? "additional_comment" : "requested_change"
button_text = approved ? "Yes, approve 2i" : "Send change request"

```

Reply:
Good suggestion @kevindew! updated